### ISSUE TITLE
Integrate MPI execution mode

### ISSUE THREAD

**Issue by JamieMair** [JamieMair/Experimenter.jl#31]
- [x] Add an option to run the job using MPI instead of Distributed.
- [ ] ~~MPI mode should be able to also handle running jobs in parallel~~

**Comment by JamieMair** [comment]
Removed running in parallel to move to a new issue.
